.github/scripts/drift_to_pr.py

```
@@ -233,7 +342,80 @@ def remote_branch_exists(branch: str, *, dry_run: bool) -> bool:
+    # one -- so fetch it and read blobs out of FETCH_HEAD rather than assuming
+    # origin/<branch> is present.
+    fetched = subprocess.run(
+        ["git", "fetch", "origin", branch],
+        check=False, text=True, capture_output=True,
+    )
+    if fetched.returncode != 0:
+        return True
+
+    for path in paths:
+        current = (REPO_ROOT / path).read_text() if (REPO_ROOT / path).exists() else None
+        previous = subprocess.run(
```

Ruff (0.16.1)

[error] `subprocess` call: check for execution of untrusted input

(S603)